crates/core/tedge_mapper/Cargo.toml

```diff
@@ -90,3 +90,4 @@ tracing = { version = "0.1", features = ["attributes", "log"] }
 
 [features]
 integration-test = []
+log_upload = ["c8y_mapper_ext/log_upload"]
```

Comment:
I would consider to remove this feature flag and to deprecate `c8y_log_plugin`.
- This will ease the build & release process: no different version of the same executable.
- This will avoid user confusion: do I need to run  `c8y_log_plugin` or `tedge_log_plugin` or both?

@reubenmiller : what's your point of view?

Reply:
I confirm that this feature flag must be removed and replaced by a dynamic configuration.

A new configuration flag will be added to enable `c8y.log-management` (defaulting to `true`) so a user can opt out if not required.